### PR TITLE
quast 3.0

### PR DESCRIPTION
**Pull request by bede**

### CODE REVIEW

File: quast.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ class Quast < Formula
   # doi "10.1093/bioinformatics/btt086"
   # tag "bioinformatics"
 
-  url "https://downloads.sourceforge.net/project/quast/quast-2.3.tar.gz"
-  sha1 "9bf176f852cf1b77f201b15e7d9262ae29cff727"
+  url "https://downloads.sourceforge.net/project/quast/quast-3.0.tar.gz"
+  sha1 "f6a72410a1bd182db733a6a3b23b79ab4e3b534f"
```
Comment (sjackman) [inline quast.rb]: Please use `sha256` rather than `sha1`.

**Comment by sjackman** [comment]
Use `brew audit --strict quast`:

```
❯❯❯ brew audit --strict quast        ⬆ ◼
==> brew style quast

1 file inspected, no offenses detected

==> audit problems
homebrew/science/quast:
 * Stable: SHA1 checksums are deprecated, please use SHA256
 * Formula should have a desc (Description).

Error: 2 problems in 1 formula
```

**Comment by bede** [comment]
I'm sorry Shaun. I was just about to PR again but saw you'd done it.

**Comment by sjackman** [comment]
No worries. Happy to help.

**Pull request closed without merging**